.cirrus.yml

```diff
@@ -0,0 +1,56 @@
+freebsd_instance:
+  image_family: freebsd-11-3-snap
+
+freebsd_build_task:
+
+  env:
+    matrix:
+      - package: npm-node10
```

Comment:
How do we expand out the matrix to the other versions we currently list as supported for FreeBSD?

Reply:
You are right, I forgot it. Back to work :)

Reply:
We can limit this to v5 also to make our lives a bit easier.

Reply:
I've been only building binaries for node version currently in the FreeBSD ports tree, which are 10, 12 and 13 now. Pushed the commit to expand the matrix.

Reply:
We can drop 10 now, but it would be good to try and get 12, 14, 16, 17 to match the other platforms